Dockerfile.standalone

```diff
@@ -0,0 +1,31 @@
+FROM node:12 AS build
+
+# This dockerfile does not require the app to be built on the host machine.
+# simply builds backstage in node image then transfers compiled files to nginx container.
+
+# You can build this image with `docker build -f Dockerfile.standalone -t backstage .`
+
+# Build backstage with node
+RUN npm install typescript -g
+
+RUN mkdir /app
+COPY . /app
+WORKDIR /app
+
+RUN yarn install && yarn tsc && yarn build
+RUN yarn workspace example-app build
```

Comment:
Should I always build example-app? I added it because It was in `package.json` script docker-build.

Reply:
It's enough to only build it, as in you should not need to run `yarn tsc` or `yarn build` above